portfolio/src/main/webapp/index.html

```diff
@@ -33,5 +33,30 @@ <h1>Hi, I'm Shreya.</h1>
       <br></br>
       <div id="list-container"></div>
     </div>
+    <div class="libraries">
+      <p class="list-header">Sentiment Score & Entity Extraction</p>
+      <p>The <strong> Natural Language API </strong> consists of Machine Learning models that understand and extract features of sentences. Very interesting stuff for the Linguist in me!</p>
+    </div>
+    <div class="sentiment">
+      <p>The ML models in the API generate a sentiment score of a sentence from -1 to 1, depending on how positive or negative the message is. Find out the score for your sentence here!</p>
+      <form action="/sentiment" method="POST">
```

Comment:
Once the servlet has a single method, I'm pretty sure the `<form>` wrapper can/should be removed, since `doSentAnalysis()` should handle everything, and we don't need the browser to do the handling that it does as part of a form submission (and it doing so may even get in the way of the js handler).